components/calendar/src/lib.rs

```diff
@@ -115,29 +115,73 @@ extern crate alloc;
 mod date;
 mod datetime;
 
+/// Types for individual calendars
+pub mod cal {
+    pub use crate::buddhist::Buddhist;
+    pub use crate::chinese::Chinese;
+    pub use crate::coptic::Coptic;
+    pub use crate::dangi::Dangi;
+    pub use crate::ethiopian::{Ethiopian, EthiopianEraStyle};
+    pub use crate::gregorian::Gregorian;
+    pub use crate::hebrew::Hebrew;
+    pub use crate::indian::Indian;
+    pub use crate::islamic::{
+        IslamicCivil, IslamicObservational, IslamicTabular, IslamicUmmAlQura,
+    };
+    pub use crate::iso::Iso;
+    pub use crate::japanese::{Japanese, JapaneseExtended};
+    pub use crate::julian::Julian;
+    pub use crate::persian::Persian;
+    pub use crate::roc::Roc;
+
+    pub use crate::any_calendar::AnyCalendar;
+
+    /// Scaffolding types: You shouldn't need to use these, they need to be public for the `Calendar` trait impl to work.
+    pub mod scaffold {
```

Comment:
Question: not sure if this should be `icu::calendar::cal::scaffold` or `icu::calendar::scaffold`. I feel like a top-level `scaffold` module is maybe better?

Reply:
Merged to avoid conflicts, I'm ambivalent here. I'd approve a PR moving this.